conda-forge.yml

```
@@ -5,4 +5,4 @@ conda_forge_output_validation: true
 conda_build:
   pkg_format: '2'
 bot:
-  automerge: true
+  inspection: hint-all
```

Please don't take initiative to change recipe configuration in upgrade PR except if needed.

```suggestion
  automerge: true
  inspection: hint-all
```